### ISSUE TITLE
Wrong group delay antenna model RNO VPol

### ISSUE THREAD

**Issue by oeyenbob** [nu-radio/NuRadioMC#397]
The group delay of the RNO Vpol is negative (~ -10ns) which is unphysical and should be fixed. Below you can see the plots picturing the behaviour of different antenna models and one can clearly see that only in the RNO Vpol the signal is seen before the computed arrival time from the ray tracer.

**Comment by boboeyen** [comment]
`
![antennamodels_arrivaltime](https://user-images.githubusercontent.com/65966730/163182870-90d68281-da9f-4e6c-8569-276183349887.png)
![antennamodels_groupdelay](https://user-images.githubusercontent.com/65966730/163182876-39bea604-506b-452c-8821-d42c4530954c.png)
`

**Comment by fschlueter** [comment]
Is this still relevant? At last it should be fixed with #492 correct?

**Comment by cg-laser** [comment]
I don't think this was ever fixed. It would require a resimulation of the Vpol response with XFDTD but I don't think Dan ever did this, or at least it didn't end up as an updated antenna model in NuRadioMC.

**Comment by fschlueter** [comment]
Ah probably I confused the group delay of the readout chain with that of the antenna itself. My bad...
